src/Symfony/Component/Intl/Data/Util/LocaleScanner.php

```diff
@@ -82,4 +82,33 @@ public function scanAliases($sourceDir)
 
         return $aliases;
     }
+
+    /**
+     * Returns all locale parents found in the given directory.
+     *
+     * @param string $sourceDir The directory with ICU files
+     *
+     * @return array An array with the locale as keys and the parent locales
+     *               as values
+     */
+    public function scanParents($sourceDir)
+    {
+        $locales = $this->scanLocales($sourceDir);
+        $fallbacks = array();
+
+        foreach ($locales as $locale) {
+            $content = \file_get_contents($sourceDir.'/'.$locale.'.txt');
+
+            // Aliases contain the text "%%PARENT" followed by the aliased locale
+            if (\preg_match('/%%Parent{"([^"]+)"}/', $content, $matches)) {
+                if ('root' === $matches[1]) {
```

Comment:
I've just understood why some have `root` as a parent: `az_Cyrl` would fall back to `az` (Azerbaijani in Latin) otherwise (so has a script change). Will update to stop `az` being a fallback in this case.